backend/models/userModel.js

```diff
@@ -17,6 +17,11 @@ const userSchema = mongoose.Schema(
       required: true,
       default: false,
     },
+    resetOtp: {
```

Comment:
Why you added this otp field in User Table?

Reply:
I will make another table to store otp.